### ISSUE TITLE
Tests on master are failing locally

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-linter#710]
```
1) Linter.extractMetadata() should flag banned JS libraries in an XPI.:
     AssertionError: expected 1 to equal 2
      at dist/webpack:/tests/test.linter.js:874:16
      at run (node_modules/core-js/modules/es6.promise.js:89:22)
      at node_modules/core-js/modules/es6.promise.js:102:28
      at flush (node_modules/core-js/modules/_microtask.js:18:9)
```

**Comment by muffinresearch** [comment]
Ah was just out of date deps. nevermind
